inference-engine/src/readers/ir_reader/ie_ir_parser.hpp

```diff
@@ -256,6 +261,41 @@ class V10Parser : public IParser {
             stringToType<double>(val, value);
             adapter.set(value);
         }
+        void on_adapter(const std::string& name, ngraph::ValueAccessor<void*>& adapter) override  {
+            std::vector<std::string> value;
+            std::vector<int64_t> shape;
+            std::string el_type_str;
+            std::vector<size_t> offset;
+            std::vector<size_t> size;
+            pugi::xml_node dn = node.child("data");
+
+            if (dn.empty())
+                THROW_IE_EXCEPTION << "No attrtibutes defined for Const op!";
```

Comment:
You don't have an information that it is a constant operation.

Reply:
Corrected.